src/blocking.rs

```diff
@@ -25,6 +25,8 @@ impl<E: fmt::Debug> std::error::Error for ReadExactError<E> {}
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 pub enum WriteFmtError<E> {
```

Comment:
```suggestion
pub enum WriteFmtError<WriteAllError<E>> {
    /// Error returned by the inner Write.
    Other(WriteAllError<E>)
```

Maybe this would be a better solution, instead of duplicating the variant?

Reply:
I think it's fine without, this way we don't nest enums too much.